config/example/rank_colors.txt

```diff
@@ -0,0 +1,11 @@
+# Use this file to denote colours for ingame admin ranks, using the following format
```

Comment:
Remove this. https://github.com/ParadiseSS13/Paradise/pull/15855 isnt merged yet.